yldio/asap-hub
# CRN 713 build test and check in github for dev 6

- fixes verify branch
- fixes variable names
- starts build step
- moves build-frontend
- adds typecheck to build
- adds sls package to build
- passes secrets to build
- reusable tests
- adds build analysis
- reusable verify
- deploy stage
- deploy squidex in deploy step
- fixes verify schema
- tries environment for build
- gets squidex secrets from environment
- moves test browser to reuable test
- fixes merge
- uses nev file for integration tests
- moves scripts
- tries more scripts
- moves Dockerfile
- setup uses action
- test job
- echo environment name
- sets squidex app name
- uses setup on build
- fixes action
- move verify-schema
- more verify
- fixes environment-name
- fixes URLs
- Initial commit of env
- sets ASAP hostname
- tries fixing verify
- frontend needs ASAP_HOSTNAME
- fixes deployment
- Remove comment
- verify first
- wire in deploy
- build frontend
- prepare Squidex action
- adds development pipeline
- Scripts to cleanup old workflows
- test retries
- image name
- fixes image name in pipeline
- fixes image name in other workflows
- updates single product
- update README
- sets branch names
- changes teardown to on-remove
- moves branch aloglia sync to pipeline
- starts moving algolia to ci pipeline
- moves branch aloglia sync to pipeline
- reusable prepare
- renames setup
- Uses outputs over env variables
